src/components/Reviews.jsx

```diff
@@ -2,19 +2,24 @@ import { useEffect, useState } from "react";
 import { getReviews } from "../api";
 import { CategoriesDropdown } from "./CategoriesDropdown";
 import { Review } from "./Review";
+import { SortByDropdown } from "./SortByDropdown";
 
 export function Reviews() {
   const [reviews, setReviews] = useState([]);
   const [chosenCategory, setChosenCategory] = useState("all");
+  const [chosenSortBy, setChosenSortBy] = useState("votes:desc");
   const [isLoading, setIsLoading] = useState(true);
 
   useEffect(() => {
     setIsLoading(true);
 
-    const urlParams = {};
+    const urlParams = new URLSearchParams();
     if (chosenCategory !== "all") {
-      urlParams.category = chosenCategory;
+      urlParams.append("category", chosenCategory);
     }
```

Comment:
React Router DOM has inbuilt search params that will also append to queries into the url.